tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -356,7 +356,15 @@ def _write_dummy_message_to_log(self, log_path, node_type):
         dummy_log_entry = "CloudWatch logs integ test - ensuring critical log file is not empty"
         self._run_command_on_head_node(f"echo '{dummy_log_entry}' > {dummy_log_message_path}")
         # Append the dummy entry to the log
-        cmd = f"sudo tee -a {log_path} < {dummy_log_message_path}"
+        log_file_owner_result = self.remote_command_executor.run_remote_command(
+            f"stat -c '%U' {log_path}", raise_on_error=False
+        )
+        if log_file_owner_result.failed:
+            # If failed, it means the `log_path` does not contain files/directories. Use root as the default owner.
+            log_file_owner = "root"
+        else:
+            log_file_owner = log_file_owner_result.stdout.strip()
```

Comment:
Looks like this is repeated, perhaps this should defined as a function? a referenced here and above?

Reply:
Done